### PR TITLE
Refactor by changing names according to correct naming conventions and spell checks

### PR DESCRIPTION
**Pull request by BenjaminKjopKofoed**
In this PR we change names of variables/methods according to correct naming conventions and spell checks to
improve the code base.

This is a part of this too large https://github.com/equinor/rvmsharp/pull/201.

### CODE REVIEW

**Review by vegasten — COMMENTED** [review]
Looks good!

There are some math heavy functions that I am not sure about. I kinda feel that they are a bit harder to read now 🤔 Not sure, though.. Math-code isn't easy to read anyway

File: CadRevealComposer/Operations/SectorSplitting/SectorSplitterOctree.cs [hunk 1]
```diff
@@ -14,9 +14,9 @@ public class SectorSplitterOctree : ISectorSplitter
     private const long SectorEstimatesTrianglesBudget = 300_000; // triangles, Arbitrary value
     private const long SectorEstimatedPrimitiveBudget = 5_000; // count, Arbitrary value
     private const float DoNotChopSectorsSmallerThanMetersInDiameter = 17.4f; // Arbitrary value
-    private const float MinDiagonalSizeAtDepth_1 = 7; // arbitrary value for min size at depth 1
-    private const float MinDiagonalSizeAtDepth_2 = 4; // arbitrary value for min size at depth 2
-    private const float MinDiagonalSizeAtDepth_3 = 1.5f; // arbitrary value for min size at depth 3
+    private const float MinDiagonalSizeAtDepth1 = 7; // arbitrary value for min size at depth 1
```
[the remark this answers is not in the document]
Comment (BenjaminKjopKofoed) [reply CadRevealComposer/Operations/SectorSplitting/SectorSplitterOctree.cs:17]: I'll revert back.

File: CadRevealComposer/Operations/ExteriorSplitter.cs [hunk 2]
```diff
@@ -14,8 +14,8 @@ namespace CadRevealComposer.Operations;
 /// Split geometries into exterior and interior. The main use is too aid sector splitting, prioritizing the exterior higher up in the sector tree.
 /// - Primitives are grouped by node ID, and whole nodes are split into exterior and interior.
 /// - Ray casting is used to measure distance from outside vantage points to the nodes. The closest nodes are kept as exterior.
-/// - TriangleMesh and InstanceMesh are tessellated to triangles and raycasted.
-/// - Other primitives are raycasted using their bounding box. The justification is that primitives have a tight bounding box.
+/// - TriangleMesh and InstanceMesh are tessellated to triangles and ray casted.
```
[the remark this answers is not in the document]
Comment (BenjaminKjopKofoed) [reply CadRevealComposer/Operations/ExteriorSplitter.cs:17]: According to [this ](https://en.wiktionary.org/wiki/ray_casting#English) it is written as **ray casting**, but I think both spellings are allowed.

**Review by Strepto — APPROVED** [review]

**Comment by Strepto** [comment]
Ty

**Pull request merged**